scripts/scripts.js

```diff
@@ -181,6 +181,7 @@ export function setLibs(location) {
   if (!['.aem.', '.hlx.', '.stage.', 'local'].some((i) => hostname.includes(i))) return '/libs';
   const branch = new URLSearchParams(search).get('milolibs') || 'main';
   if (branch === 'local') return 'http://localhost:6456/libs';
+  if (branch === 'main') return `${window.location.origin}/libs`;
```

Comment:
What about when you're on `main--da-bacom--adobecom.aem.live`? Then it would return `main--da-bacom--adobecom.aem.live/libs`, right? Your "after" link 404s so I can't verify.

Seems like we should either remove `'.stage.'` from the check in line 181 (which would also prevent us from using the milolibs param on stage) or we need to add a check to the if statement that the hostname equals the stage hostname.

@JasonHowellSlavin @Brandon32 Thoughts?

Reply:
We need a hostname check after the search param.
```suggestion
  if (hostname.includes('.stage.' && branch === 'main') return `${window.location.origin}/libs`;
```

@robert-bogos Can you make sure your repo is synced with the [AEM Code Sync GitHub App](https://github.com/apps/aem-code-sync), thanks!